models/chainlink/arbitrum/chainlink_arbitrum_ccip_fulfilled_transactions.sql

```diff
@@ -2,61 +2,36 @@
   config(
     
     alias='ccip_fulfilled_transactions',
-    partition_by=['date_month'],
+    partition_by=['date_start'],
```

Comment:
is the dataset large enough to justify making partitions much more granular (day vs. month)?

Reply:
this was done only since I have taken out date_month because it was relevant anymore but if it makes sense to add it back then I can do so. Its not terribly large.

Reply:
removing partition altogether is fine too, if you have no need for monthly column and dataset isn't massive